extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -100,7 +100,9 @@ export abstract class AzureAuth implements vscode.Disposable {
 				}
 			}
 			Logger.error(ex);
-			return undefined;
+			return {
+				canceled: false
```

Comment:
Shouldn't we be rejecting the loginComplete promise as well?

Reply:
good catch